docs/source/troubleshooting/general.rst

```diff
@@ -37,6 +37,43 @@ Also, if you have many controllers already directly connected to the host, it mi
 Sunshine provided controller (connected to the guest) is the "first" one. In Linux this can be accomplished on USB
 devices by finding the device in `/sys/bus/usb/devices/` and writing `0` to the `authorized` file.
 
+Network performance test using iPerf3
+-------------------------------------
+For real-time game streaming the most important characteristic of the network
+path between server and client is not pure bandwidth but rather stability and
+consistency (low latency with low variance, minimal or no packet loss).
+
+The network can be tested using the multi-platform tool iPerf3_.
+
+.. _iPerf3: https://iperf.fr
+
+On the Sunshine host ``iperf3`` is started in server mode:
+
+.. code-block:: bash
+
+   iperf3 -s
+
+On the client device iperf3 is asked to perform a 60-second UDP test in reverse
+direction (from server to client) at a given bitrate (e.g. 50 Mbps):
+
+.. code-block:: bash
+
+   iperf3 -c HostAddress -t 60 -u -R -b 50M
```

Comment:
```suggestion
   iperf3 -c {HostIpAddress} -t 60 -u -R -b 50M
```

I think this will make it more clear that the host address needs to be replaced.